app/views/chapter/show.html.haml

```diff
@@ -1,28 +1,28 @@
-%section
-  .row.chapter-header
-    .medium-6.columns.chapter-header-text
+.container
+  .row
+    .col-12.col-md-6
       %h1= @chapter.name
       - if @chapter.description
         %p.lead= @chapter.description
       - else
         %p.lead= t('chapters.intro_html')
     - if @chapter.image.present?
-      .medium-6.columns.chapter-header-image
+      .col-12.col-md-6
         = image_tag @chapter.image.bg, alt: "#{@chapter.name} chapter"
 
-.panel
+.alert.alert-primary
```

Comment:
@despo I use alert here. It's not technically an alert but it looks like that component. Basically a div with coloured background and border.

Reply:
👍